src/TestExplorer/TestExplorer.ts

```diff
@@ -168,16 +168,16 @@ export class TestExplorer {
      * Discover tests
      */
     async discoverTestsInWorkspace() {
-        const toolchain = this.folderContext.workspaceContext.toolchain;
-        if (toolchain.swiftVersion.isLessThan(new Version(5, 11, 0))) {
-            await this.discoverTestsInWorkspaceSPM();
-        } else {
+        //const toolchain = this.folderContext.workspaceContext.toolchain;
+        //if (toolchain.swiftVersion.isLessThan(new Version(5, 11, 0))) {
```

Comment:
I'm assuming this works fine on Swift 6 builds (or at least doesn't break anything)?

Reply:
It used the custom LSP test listing requests from Swift 6, but these have changed structure and the code needs rewritten so I have temporarily disabled the Swift 6 code path. This LSP request has not been in a release yet so we aren't removing anything